installer/install_scripts/sshprep

```diff
@@ -230,8 +230,8 @@ check_config_block() {
 			prepend_config_block "\nHost ${2} ${local_ip}\n\tHostname\t\t${local_ip}\n\tHostKeyAlias\t\t${2}\n${user_line}\n"
 		else
 			status "No user-supplied target IP or hostname for $2, look one up"
-			ip4=$(dig +short ${2} A)
-			ip6=$(dig +short ${2} AAAA)
+			ip4=$(dig +nocomment +short ${2} A 2>/dev/null | sed -e 's/;;.*//' | grep -v '^$')
+			ip6=$(dig +nocomment +short ${2} AAAA 2>/dev/null | sed -e 's/;;.*//' | grep -v '^$')
```

Comment:
A few things here:

1. Should we handle cases where one hostname corresponds to multiple IP addresses? There are a few cases that I'm aware of in which this could happen and it might cause commands to fail. If for some reason we needed to treat each IP address as different, we could possibly handle it by iterating over the value of `ip4` and `ip6` variables. However, I think in most cases that this could happen, we would be better off just picking one IP address.
2. To my knowledge, adding `| sed -e 's/;;.*//' | grep -v '^$'` is made redundant by the combination of `+nocomment` and  `+short` dig arguments. Unless I'm mistaken, we could probably cut those.

Reply:
1. You are correct - the Hostname field where $ipv4 and/or $ipv6 are used only supports a single address.  I agree, limiting this to a single IP makes sense.  Would you please add ```| head -1``` after the grep -v '^$' on both the ipv4 and ipv6 lines and before the right parenthesis?
2. I thought that was true as well and that's why the first version did not have the sed and grep commands.  Unfortunately I was wrong and these have to be part of the lookup.

Reply:
I've updated the file in the update-sshprep branch (to include head -1 for both the ipv4 and ipv6 variable assignments.  It's not clear whether the change propagated to the PR or not.

Reply:
Thanks for making that change. For future reference, any pushes you make to a branch after PR will be associated with that PR. Makes for easy response to change requests.

Just curious, can you provide a scenario that you found during testing where having `+nocomment +short` still resulted in empty lines and comments (`;;`)? Thanks.